src/components/PageHeader/_page-header.scss

```diff
@@ -28,34 +20,22 @@
   }
 
   .title-banner__standfirst {
-    padding-top: rem(16);
-
-    @include mq(sm) {
-      padding-top: rem(24);
-    }
-
-    @include mq(md) {
-      padding-top: rem(32);
-    }
+    padding-top: var(--space-lg);
   }
 }
 
 .title-banner__title {
   margin: 0;
   // @todo commenting out for now. mixin need to be pulled in.
   // @include headingDivider((var(--text-base-size-px) / 4), ($grid-spacing-unit * 2.5));
-  padding: 0 0 rem(16);
+  padding-bottom: calc(2 * var(--space-unit));
 }
 
 .title-banner__standfirst {
-  padding-bottom: rem(32);
+  padding-bottom: var(--space-xl);
   text-align: center;
 
-  @include mq(sm) {
-    padding-bottom: rem(32);
-  }
-
-  @include mq(md) {
-    padding-bottom: rem(64);
+  @include mq($from: sm, $until: md) {
```

Comment:
I'm guessing this is an override for --space-xl. Title banner is one of the more recent components which would almost certainly use the new spacings so I'd remove this and let the variable height work as normal (unless it looks wrong?)

Reply:
Sounds good, I'll do a quick visual check to make sure